### PR TITLE
refactor: Removing test for all tabs 

### PR DESCRIPTION
**Pull request by rajsam003**
This PR resolves []

### PR Checklist

- [x] I tested my changes on **iOS** and **Android**.
- [ ] I added screenshots or videos to illustrate my changes.
- [ ] I added Tests and Stories for my changes.

### To the reviewers 👀

- [ ] I would like **at least one** of the reviewers to run this PR on the simulator or device.

Need help with something? Have a look at our [docs], or get in touch with us.

[docs]: /docs/README.md

### CODE REVIEW

File: src/Screens/HomeTabs/Artists/ArtistTabs/ArtistTabs.tsx [hunk 1]
```diff
@@ -31,8 +32,12 @@ const Header = ({ artistName, navigation }: TabBarProps & HeaderProps) => {
...
 }
 
-export const ArtistTabs = ({ route, navigation }: ArtistTabsProps) => {
+export const ArtistTabs = () => {
   const insets = useSafeAreaInsets()
+
+  const route = useRoute<ArtistTabsProps["route"]>()
+  const navigation = useNavigation<ArtistTabsProps["navigation"]>()
+
   const { artistName } = route.params
```
Comment (pvinis) [inline src/Screens/HomeTabs/Artists/ArtistTabs/ArtistTabs.tsx:41]: are we doing slug here?

Comment (rajsam003) [reply src/Screens/HomeTabs/Artists/ArtistTabs/ArtistTabs.tsx:41]: I tried with slug and with that, we use the query to get just the artist name to display as the header. So for that, we need to `useLazyLoadQuery` and `Suspense` which I think is a lot of work. So I thought it simple to just pass `artistName` in a `route.param`. What do you think?

Comment (rajsam003) [reply src/Screens/HomeTabs/Artists/ArtistTabs/ArtistTabs.tsx:41]: Also I am getting this annoying issue even though I am using `<Suspense fallback={<ActivityIndicator />}>`

image

Comment (rajsam003) [reply src/Screens/HomeTabs/Artists/ArtistTabs/ArtistTabs.tsx:41]: https://stackoverflow.com/questions/54432861/a-react-component-suspended-while-rendering-but-no-fallback-ui-was-specified

Comment (pvinis) [reply src/Screens/HomeTabs/Artists/ArtistTabs/ArtistTabs.tsx:41]: lets merge for now, and we can figure this out afterwards.

for sure we need to pass the slug, so we can actually do the relay query, but lets not block this PR.

Comment (rajsam003) [reply src/Screens/HomeTabs/Artists/ArtistTabs/ArtistTabs.tsx:41]: @MounirDhahri helped me with the solution to use slug 🙏🏽. I will use it on my next PR.

**Pull request merged**